closed/src/java.base/share/classes/openj9/internal/security/RestrictedSecurityConfigurator.java

```diff
@@ -37,6 +37,10 @@
 
 import sun.security.util.Debug;
 
+/*[IF CRIU_SUPPORT]*/
+import openj9.internal.criu.InternalCRIUSupport;
+/*[ENDIF] CRIU_SUPPORT*/
```

Comment:
`RestrictedSecurityConfigurator.java` might need to be added to `GensrcJ9JCL.gmk`?

Reply:
No, all `closed` source files are to be JPP processed.